src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -74,7 +74,12 @@ const ContentTagsDrawer = ({ id, onClose }) => {
     data: contentTaxonomyTagsData,
     isSuccess: isContentTaxonomyTagsLoaded,
   } = useContentTaxonomyTagsData(contentId);
-  const { data: taxonomyListData, isSuccess: isTaxonomyListLoaded } = useTaxonomyList(org);
+
+  // Taxonomies by Organization
+  const { data: taxonomyListData, isSuccess: isTaxonomyListLoaded } = useTaxonomyList(org, contentId);
+
+  // All taxonomies to verify if exists object tags in other taxonomy that does not belong to the organization.
+  const { data: allTaxonomyListData, isSuccess: isAllTaxonomyListLoaded } = useTaxonomyList(undefined, contentId);
```

Comment:
Hi @ChrisChV !
This call (even if we don't explicitly include an Org) will filter out the taxonomies on which the user has no permission (i.e. taxonomies from other/no orgs).

I'm getting it wrong here?

Reply:
In the [testing](https://www.loom.com/share/5e87a1249004437d869836d691cdb26e?sid=9e42e2cc-cf25-4a8e-9514-113fe5d6c034) I have done, the Test taxonomy does not belong to any organization. So currently it doesn't filter those permissions :thinking:

Reply:
I think we have two different cases:
1 - In the one you show, the user have permission to access the taxonomy (because he is the staff?).
2 - The user is not staff (so he can't see no-org taxonomies)

We have 1 covered. Do we need to worry about 2?